### ISSUE TITLE
Unable to navigate in p-tree when using NVDA or JAWS screen reader

### ISSUE THREAD

**Issue by JirkaBures** [primefaces/primeng#8674]
Hello, the title says it all.

You can try on primeng demo pages https://primefaces.org/primeng/showcase/#/tree

I would like to use p-tree with selectionMode="checkbox". But problem is with every mode.

When i try navigating in tree with keyboard arrow keys **without screen reader active** it seems to work. But **when screen reader is active** the arrow keys dont work.

**Comment by JirkaBures** [comment]
fixed by moving role="treeitem" to div.ui-treenode-content, and reseting role on LI to role="none"
inspired by https://www.w3.org/TR/wai-aria-practices/examples/treeview/treeview-2/treeview-2a.html
I did it the "filthy" way


`const liElements = this.pTree.el.nativeElement.getElementsByClassName('ui-treenode');
      for (const liEl of liElements) {
        liEl.setAttribute('role', 'none');
      }
      const divElements = this.pTree.el.nativeElement.getElementsByClassName('ui-treenode-content');
      for (const divEl of divElements) {
        divEl.setAttribute('role', 'treeitem');
      }`


`onNodeExpand() {
    setTimeout(() => {
      this.fixTreeHtmlRole();
    });
  }`
